rllib/env/multi_agent_env_runner.py

```diff
@@ -815,12 +804,24 @@ def make_env(self):
                     check_multiagent_environments(env.unwrapped)
             except Exception as e:
                 logger.exception(e.args[0])
-        # If not required, still check the type (must be `MultiAgentEnv``).
+        # If not required, still check the type (must be `VectorMultiAgentEnv``).
         else:
-            assert isinstance(self.env.unwrapped, MultiAgentEnv), (
-                "ERROR: When using the `MultiAgentEnvRunner` the environment needs "
-                "to inherit from `ray.rllib.env.multi_agent_env.MultiAgentEnv`."
-            )
+            try:
+                assert isinstance(self.env, VectorMultiAgentEnv)
+                assert isinstance(self.env.envs[0].unwrapped, MultiAgentEnv)
+            except AssertionError:
+                logger.exception(
+                    "When using the `MultiAgentEnvRunner`, the environment must "
+                    f"inherit from `ray.rllib.env.vector.vector_multi_agent_env."
+                    f"VectorMultiAgentEnv` (but yours is {self.env}) and the individual"
```

Comment:
This is imo misleading when the `VectorMultiAgentEnv` is used but the `MultiEnv` is not which - I think - could practically happen. Maybe better separating the assertions and messages?